config/settings/development.yml

```diff
@@ -0,0 +1,6 @@
+data:
+  sync_asset_path: "C:/Users/sureshs/Visual Studio/Visual Studio Code Repo/ogipilot-demo-run/ogipilot-demo-run/data/SyncAssets.xml.erb"
```

Comment:
I would normally try to avoid committing absolute paths to a repository that might have personal info in them.